src/plugins/resolver/resolver.c

```diff
@@ -73,6 +73,9 @@ static void resolverInit (resolverHandle * p, const char * path)
 	p->tempfile = 0;
 
 	p->path = path;
+
+	p->uid = getuid ();
+	p->gid = getgid ();
```

Comment:
Those 2 lines are just to initialize uid and gid, they get set the the actuall files owner in after the stat call in resolver, get